### ISSUE TITLE
Improve etnservice function documentation and examples

### ISSUE THREAD

**Issue by PietrH** [inbo/etnservice#31]
Currently the function documentation (roxygen) within `etnservice` is the same as it was on `etn`, except for the adapted parameter `credentials` which replaces `connection`. The examples haven't been changed at all (so will not work). 

This documentation is currently not user facing, as etn uses it's own sql helper and roxygen documentation (see inbo/etn#280). Meaning that the database interfacing code needs to be maintained in two places. However, we are considering reducing this to one place by making `etnservice` a dependency of `etn` and using/evaluating/parsing the code from etnservice for use in the database interfacing functions of `etn`.

In short, either we should update the documentation in `etnservice` to be more userfriendly for future collaborators, or we should update it to be correct and client facing once more, so it can be inherited by `etn`.
